### DISCUSSION TITLE
React + NodeJs Ou React + Java

### DISCUSSION THREAD

**Discussion by andersonbuenos** [frontendbr/forum#2435] [Discussão]
Prezados boa noite! Estou pra finalizar o curso de ReactJS e disponível no meu curso tem Java e Node. Como iniciante pensa na dúvida da peste. Gostaria de saber sobre o mercado o que ele tá pedindo mais hoje, qual é mais sólido no mercado? Obrigado

**Comment by ApenasGabs** [comment]
Rapaz, solidas essas stacks são, a questão é que na data de hoje o mercado ta meio chato para iniciantes, mas acho super valido você passar pelos dois, pegar uns projetos mais práticos e fazer tambem no final de cada curso

**Reply by andersonbuenos** [reply]
Hoje estou como analista de BI, a ideia é não ficar parado porque o mercado muito e gosto de muito de programar. Muito obrigado pela dica vou fazer os dois. 😁
